liquibase-standard/src/main/java/liquibase/snapshot/SnapshotGeneratorFactory.java

```diff
@@ -447,4 +469,35 @@ private void getContainerTypes(Class<? extends DatabaseObject> type, Database da
         }
 
     }
+
+    private void syncBackingIndex(PrimaryKey pk, DatabaseSnapshot snapshot) {
```

Comment:
Can we combine these two method? They look very similar.

Reply:
I was able to refactor to 1 method by using the underlying attributes map.